core/tests/ts-integration/tests/contracts.test.ts

```diff
@@ -423,35 +423,6 @@ describe('Smart contract behavior checks', () => {
         expect(receipt.status).toEqual(1);
     });
 
-    test('Should check transient storage', async () => {
```

Comment:
Are those tests removed because of `etc/contracts-test-data` removal? If so then I would add contracts to `ts-integration/contracts` and keep the tests

Reply:
I've moved the relevant 2 tests to the `state_keeper::executor` module, so they work as unit tests now.